### DISCUSSION TITLE
Pomegrenade plants are TOO EXPLOSIVE!?

### DISCUSSION THREAD

**Discussion by diegoyocys** [FakeFishGames/Barotrauma#5294] [Ideas and suggestions]
hello!

I have grown a pomegrenade plant, but I find them that the probability of a fire is too high? sometimes I have to put out 3 fires in less than 10mins, sometimes 2 fires in less than 2 mins. while docked into an outpost sometimes im afraid that a fire might grow to big inside the sub while unattended?

If you think the reward is worth the risk, at least consider designing a conical basket (that can be constructed) to collect all pomegrenades below the plant all at once (that can be used for other plants too), so you can retrieve them quickly, instead of picking them one at the time with the chance of blowing you up (which happened twice to me)

An entire another option is to leave the probability of fire as it is, and no conical basket, but instead of an explosion, a fire that develops just as big as the explosion, but over... lets say... 3 seconds?, that way gives you the chance to run for safety.

what do you think?

**Comment by Krzeszny** [comment]
Fire instead of explosion sounds like a good idea. A bigger problem is that welding fuel tanks explode instantly when on fire.
